packages/web/src/lib/txn.ts

```diff
@@ -1,12 +1,17 @@
-import algosdk, { SuggestedParams, Transaction } from 'algosdk';
+import algosdk, { SuggestedParams, Transaction } from "algosdk";
 
-import { BuilderError } from '../errors/errors';
-import { ERRORS } from '../errors/errors-list';
+import { BuilderError } from "../errors/errors";
+import { ERRORS } from "../errors/errors-list";
 import { AccountAddress, ExecParams, SignType, TransactionType, TxParams } from "../types";
 import { parseAppArgs } from "./parsing";
 
-export function encodeNote (note: string | undefined, noteb64: string| undefined): Uint8Array | undefined {
-  if (note === undefined && noteb64 === undefined) { return undefined; }
+export function encodeNote(
+  note: string | undefined,
+  noteb64: string | undefined
+): Uint8Array | undefined {
+  if (note === undefined && noteb64 === undefined) {
+    return undefined;
+  }
```

Comment:
I think you have some extra auto formatting.

Reply:
yes, I am using prettier, will disable it